README.md

```diff
@@ -42,8 +38,7 @@ jobs:
         uses: A5100907/branch-name-enforcer@v1
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          ignore_branches: "master;main;teamshare;product_teamshare"
-          regex: "^([A-Z0-9]*/)((teamshare)|(feature/[A-Z]*-[0-9]*)|(release/[0-9]{1,2}.[0-9]{1,2}.[0-9]{1,5}$))"
+          regex: "^(master)$|^([A-Z0-9]*/)((teamshare)|(feature/[A-Z]*-[0-9]*)|(release/[0-9]{1,2}.[0-9]{1,2}.[0-9]{1,5}$))"
```

Comment:
([A-Z0-9]*/)((teamshare) 
can also be written where the project name is hardcoded. For example,
(my_project/)((teamshare)...